### PR TITLE
Feature/df 2145/edit object in new window

### PR DESCRIPTION
**Pull request by ganqqwerty**

### CODE REVIEW

File: alv-portal-ui/src/app/competence-catalog/ch-fiches/ch-fiche/ch-fiche.component.html [hunk 1]
```diff
@@ -394,3 +433,20 @@
     <fa-icon [icon]="['fas', 'search-plus']"></fa-icon>
   </button>
 </ng-template>
+<ng-template #editObjectInNewWindowButton
+             let-className="class"
+             let-objectId='objectId'
+             let-path="path"
+>
+  <a [ngClass]="className"
+     [ngbTooltip]="'edit-in-new-window' | translate"
```
Comment (emina-str) [inline alv-portal-ui/src/app/competence-catalog/ch-fiches/ch-fiche/ch-fiche.component.html:442]: add translation, at least a placeholder.

File: alv-portal-ui/src/app/competence-catalog/shared/competence-element-modal/competence-element-modal.component.html [hunk 2]
```diff
@@ -41,6 +41,15 @@
...
   </ng-container>
-
+  <ng-container *alvHasAnyAuthority="[userRole.ROLE_KK_EDITOR, userRole.ROLE_ADMIN]">
+    <a *ngIf="isReadonly && competenceElementId"
+       [routerLink]="['/', 'kk', 'competence-elements', 'edit', competenceElementId]"
+       actionButton
+       class="btn btn-text mr-0 mr-md-1 mb-1 mb-md-0"
+       target="_blank"
+       type="button">
+      {{'edit-in-new-window' | translate}}
```
Comment (emina-str) [inline alv-portal-ui/src/app/competence-catalog/shared/competence-element-modal/competence-element-modal.component.html:51]: as before, i would suggest to write a label placeholder, at least.

File: alv-portal-ui/src/app/competence-catalog/shared/competence-set/competence-set.component.html [hunk 3]
```diff
@@ -181,3 +199,17 @@
   </ng-container>
 </ng-template>
 
+<ng-template #editCompetenceElementInNewWindowButton
+             let-elementId='elementId'>
+  <a [ngbTooltip]="'edit-in-new-window' | translate"
```
Comment (emina-str) [inline alv-portal-ui/src/app/competence-catalog/shared/competence-set/competence-set.component.html:204]: :) here also

File: alv-portal-ui/src/app/competence-catalog/competence-elements/competence-elements-resolver.service.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import { ActivatedRouteSnapshot, Resolve, Router, RouterStateSnapshot } from '@angular/router';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CompetenceElementsResolverService implements Resolve<boolean> {
```
Comment (emina-str) [inline alv-portal-ui/src/app/competence-catalog/competence-elements/competence-elements-resolver.service.ts:7]: do we use this anywhere?

File: alv-portal-ui/src/app/competence-catalog/ch-fiches/ch-fiche/ch-fiche.component.html (hunk 1 above)
Comment (ganqqwerty) [reply alv-portal-ui/src/app/competence-catalog/ch-fiches/ch-fiche/ch-fiche.component.html:442]: oops, forgot

File: alv-portal-ui/src/app/competence-catalog/competence-elements/competence-elements-resolver.service.ts (hunk 4 above)
Comment (ganqqwerty) [reply alv-portal-ui/src/app/competence-catalog/competence-elements/competence-elements-resolver.service.ts:7]: no, will delete it. I use guards now

**Pull request merged**